internal/portforward/service/command.go

```diff
@@ -0,0 +1,32 @@
+package service
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func (s *Service) runUpCommand(ctx context.Context, ports []uint16) (err error) {
+	// run command replacing {{PORTS}} with the ports (space separated)
```

Comment:
any better suggestion than using CSV to seperate values? That could be added in #1113 !